Spec/Auth.swift

```diff
@@ -1717,14 +1718,12 @@ class Auth : QuickSpec {
                     waitUntil(timeout: testTimeout) { done in
                         auth.authorise(nil, options: authOptions) { tokenDetails, error in
                             expect(authCallbackHasBeenInvoked).to(beTrue())
-
-                            authCallbackHasBeenInvoked = false
```

Comment:
> No, we need to test that the defaults are used for subsequent authorisations so if we passed in arguments then we'd be changing those defaults?

I might be missing something, but as per `RSA10j` spec that states: 
>For example, if a client is initialised with `TokenParams#ttl` configured with a custom value, and a `TokenParams` object is passed in as an argument to `authorise` with a null value for `ttl`, then the `ttl` used for every subsequent authorisation will be null. 

Does not this mean that the default parameters need to be replaced?